src/main/java/dan200/computercraft/shared/peripheral/speaker/SoundTask.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
The issue with this is that I can't use a constructor in an anonymous class, which is currently how I pass the args initially.